include/download_client.h

```diff
@@ -141,13 +153,12 @@ int download_client_init(struct download_client *client,
  *
  * @param[in] client	Client instance.
  * @param[in] host	HTTP server to connect to, null-terminated.
- * @param[in] sec_tag	TLS credentials tag, used for HTTPS connections.
- * 			Must be -1 when using HTTP.
+ * @param[in] config	TLS and APN configuration options.
  *
  * @retval int Zero on success, a negative error code otherwise.
  */
 int download_client_connect(struct download_client *client, char *host,
-			    int sec_tag);
+			    struct download_client_cfg *config);
```

Comment:
We should probably make the input parameters `const` pointers?

Reply:
Right, done.